doc/tutorial/invoking-zeekctl.rst

```diff
@@ -1,9 +1,218 @@
+   manager      manager localhost        4394    1013M  154M   0%  zeek
+   proxy-1      proxy   localhost        4446    1013M  153M   0%  zeek
+   worker-1     worker  localhost        4511      1G   279M   0%  zeek
+   worker-2     worker  localhost        4512      1G   279M   0%  zeek
+
+This command is different in the "interactive" mode of ``zeekctl`` - try
+running ``zeekctl`` alone, then ``top``. Here you can watch the various
+nodes' resources live!
+
+You can also check the status of your nodes with ``zeekctl status``:
+
+.. code:: console
+
+   # zeekctl status
+   Name         Type    Host             Status    Pid    Started
+   logger-1     logger  localhost        running   4341   08 Dec 21:47:32
+   manager      manager localhost        running   4394   08 Dec 21:47:33
+   proxy-1      proxy   localhost        running   4446   08 Dec 21:47:34
+   worker-1     worker  localhost        running   4511   08 Dec 21:47:35
+   worker-2     worker  localhost        running   4512   08 Dec 21:47:35
+
+This is useful to ensure the nodes are healthy and running. 
+
+If you see that a certain node is stopped via ``zeekctl status`` and need
+to restart just that node, you can specify it as an argument. For example,
+here we kill the ``proxy-1`` node, then start just that node after killing
+it:
+
+.. code:: console
+
+   # kill 4446
+   # zeekctl status
+   Name         Type    Host             Status    Pid    Started
+   logger-1     logger  localhost        running   4341   08 Dec 21:47:32
+   manager      manager localhost        running   4394   08 Dec 21:47:33
+   proxy-1      proxy   localhost        crashed
+   worker-1     worker  localhost        running   4511   08 Dec 21:47:35
+   worker-2     worker  localhost        running   4512   08 Dec 21:47:35
+   # zeekctl start proxy-1
+   starting proxy ...
+
+   # zeekctl status
+   Name         Type    Host             Status    Pid    Started
+   logger-1     logger  localhost        running   4341   08 Dec 21:47:32
+   manager      manager localhost        running   4394   08 Dec 21:47:33
+   proxy-1      proxy   localhost        running   4947   08 Dec 21:53:49
+   worker-1     worker  localhost        running   4511   08 Dec 21:47:35
+   worker-2     worker  localhost        running   4512   08 Dec 21:47:35
+
+This requires manually intervention. Instead, you can use ``zeekctl`` with
+``cron`` in order to automatically check for crashed nodes and restart them.
+See the `ZeekControl cron`_ command reference for more information.
+
+Many basic clusters can be maintained with just what was discussed here:
+a mixture of ``zeekctl`` commands like ``deploy``, ``start``, ``stop``,
+and ``status``. You can use ``top`` and ``cron`` for extra visibility
+or configuration on your cluster. Then, configure the cluster according
+to your particular environment, and that should be what you need to get
+a capable Zeek cluster.
```

Comment:
For examining if anything is off,  `zeekctl diag [node]` is the one command to use. Might be worth including. It shows `.stderr` and also `reporter.log` by default IIRC.